spec/features/purchasing_a_reservation_on_behalf_of_spec.rb

```diff
@@ -46,8 +46,8 @@
       it "can purchase" do
         click_link instrument.name
 
-        # time is frozen to 9:30am, we expect the default time to be the end of the admin reservation
-        expect(page.find_field("reservation_reserve_start_date").value).to match(%r[09/11/\d{4}])
+        # time is frozen to 9:30am ten days after fiscal year start, we expect the default time to be the end of the admin reservation
+        expect(page.find_field("reservation_reserve_start_date").value).to eq Date.today.strftime("%m/%d/%Y")
```

Comment:
I think this is ok, but are we sure that `Date.today` is timezone aware? I always forget.

Reply:
The internet says `Date.today` uses server time, not application time. Should I switch to `Time.zone.now.to_date`, or is there a better way?

Reply:
I think `Time.zone.today` should be fine.